src/lib.rs

```diff
@@ -69,6 +71,9 @@ pub struct Ammonia<'a> {
     pub url_schemes: HashSet<&'a str>,
     /// Permit relative URLs on href and src attributes.
     pub url_relative: bool,
+    /// Stick these rel="" attributes on every link.
+    /// If rel is in the generic or tag attributes, this must be `None`.
+    pub link_rel: Option<&'a str>,
```

Comment:
Yep, I like that, this is way better than trying to handle thing like is it a link and is rel not blacklisted and hardcode noopener noreferrer (That's what I started doing but I threw it away because it was way to messy...)